src/EditorFeatures/Core/Shared/Tagging/TagSources/ViewTagSource.cs

```diff
@@ -20,11 +20,12 @@ internal class ViewTagSource<TTag> : ProducerPopulatedTagSource<TTag> where TTag
         private readonly ITextView _textView;
 
         public ViewTagSource(
-            ITextView textView, ITextBuffer subjectBuffer, ITagProducer<TTag> tagProducer, ITaggerEventSource eventSource,
-            IAsynchronousOperationListener asyncListener, IForegroundNotificationService notificationService,
-            bool removeTagsThatIntersectEdits,
-            SpanTrackingMode spanTrackingMode) :
-            base(subjectBuffer, tagProducer, eventSource, asyncListener, notificationService, removeTagsThatIntersectEdits, spanTrackingMode, bufferToRelatedTagSource: null)
```

Comment:
We go from 8 parameters to 5 across the board.  tagProducer/eventSource/removeTagsThatIntersectEdits/spanTrackingMode are all absorbed into ```dataSource```